release/serve_tests/serve_tests.yaml

```diff
@@ -0,0 +1,12 @@
+- name: single_deployment_1k_noop_replica
+  cluster:
+    app_config: app_config.yaml
+    compute_template: compute_tpl_8_cpu.yaml
+
+  run:
+    timeout: 10800
```

Comment:
This time out is too long for nightly test, maybe just 1hr?

Reply:
im thinking to give it some buffer since by default 10 * 10m trials already takes >1.5hrs, plus about 0.3~0.5hr for autoscaler, my previous value of 2hrs is a bit too tight and might introduce flakiness. It should be totally doable to make it 1hr tho but it requires running with less trials / less replica, etc. We can iterate on this for sure.

Reply:
yeah if we are running nightly, maybe we should be 5*5m trials instead so end to end test can finish in <1.5hr